### PR TITLE
release note for azure cli 2.13.0

### PR DESCRIPTION
**Pull request by azclibot**

### CODE REVIEW

**Review by jiasli — APPROVED** [review]

File: docs-ref-conceptual/includes/current-version.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 author: dbradish-microsoft
 ms.author: dbradish
 manager: barbkess
-ms.date: 09/29/2020
+ms.date: 10/20/2020
```
Comment (jiasli) [inline docs-ref-conceptual/includes/current-version.md:5]:
```suggestion
ms.date: 10/13/2020
```

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ description: Learn about the latest updates to Azure CLI
 author: dbradish-microsoft
 ms.author: dbradish
 manager: barbkess
-ms.date: 09/29/2020
+ms.date: 10/20/2020
```
Comment (jiasli) [inline docs-ref-conceptual/release-notes-azure-cli.md:7]:
```suggestion
ms.date: 10/13/2020
```

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 3]
```diff
@@ -15,6 +15,104 @@ ms.custom: devx-track-azurecli
 
 # [Current release notes](#tab/azure-cli)
 
+## October 20, 2020
```
Comment (jsntcy) [inline docs-ref-conceptual/release-notes-azure-cli.md:18]: The date is not correct, and it should be October 13, 2020

Comment (jiasli) [inline docs-ref-conceptual/release-notes-azure-cli.md:18]:
```suggestion
## October 13, 2020
```

Comment (jiasli) [reply docs-ref-conceptual/release-notes-azure-cli.md]: The CI uses the next Tuesday by default. However, we were blocked last Friday.

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 4]
```diff
@@ -15,6 +15,104 @@ ms.custom: devx-track-azurecli
...
+* Support virtual-node with aks create: `az aks create --enable-addons virtual-node`
+* Add node image only option for CLI
+* Expect kube-dashboard addon be disabled by default
+* `az aks create/update`: Add LicenseType support for Windows
+* Support add Spot node pool
+* Honor addon names defined in Azure CLI
+
+### AMS
+
+* Fix #14687: Mixed resource group and account name in command "az ams streaming-endpoint show" #14687
```
Comment (fengzhou-msft) [inline docs-ref-conceptual/release-notes-azure-cli.md:38]:
```suggestion
* Fix #14687: Mixed resource group and account name in command "az ams streaming-endpoint show"
```

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**